__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -433,29 +433,73 @@ export const Error: Story = {
 };
 
 /**
- * With start icon, you can customize the icon that appears at the beginning of
+ * With `startIcon`, you can customize the icon that appears at the beginning of
  * the Combobox. This is useful when you want to add a custom icon to the
  * component.
  *
- * In this example, we show how this is done by setting the `startIcon` prop.
+ * **NOTE:** When `startIcon` is set, we set some default values for the icon:
+ * - `size`: "small"
+ * - `color`: `semanticColor.icon.default`
+ *
+ * You can customize the size and color of the icon by passing the `size` and
+ * `color` props to the `PhosphorIcon` component.
  */
 export const StartIcon: Story = {
     render: function Render(args: PropsFor<typeof Combobox>) {
         const [_, updateArgs] = useArgs();
 
         return (
             <View style={{gap: spacing.medium_16}}>
-                <LabelMedium>Default:</LabelMedium>
+                <LabelMedium>With default size and color:</LabelMedium>
                 <Combobox
                     {...args}
+                    startIcon={
+                        <PhosphorIcon icon={magnifyingGlassIcon} size="small" />
```

Comment:
**suggestion:** We can remove `size="small"` here so we can confirm that the default size is being applied!

Reply:
Good catch! I'll fix it before landing.